autoload/neobundle/commands.vim

```diff
@@ -485,10 +485,16 @@ function! neobundle#commands#save_cache() "{{{
         \ v:progname, current_vim, string(bundles)], cache)
 endfunction"}}}
 function! neobundle#commands#load_cache(...) "{{{
-  let vimrc = get(a:000, 0, $MYVIMRC)
   let cache = neobundle#commands#get_cache_file()
-  if !filereadable(cache) || getftime(cache) < getftime(vimrc)
-    return 1
+  if !filereadable(cache) | return 1 | endif
+
+  let vimrc = get(a:000, 0, $MYVIMRC)
```

Comment:
I think it should be `let vimrcs = get(a:000, 0, [$MYVIMRC])`.
Below if statement is not needed.